Automattic/jetpack
# Contact Info block: Update to include margins design tool

The proposed change updates the `Contact info` block with an ability for a user to specify margins for the block in the dimensions section of its design tools.

![Screenshot 2022-02-22 at 13 59 30](https://user-images.githubusercontent.com/2019970/155149313-62839ec1-8cb6-49fa-8d41-ae8f240ca06b.png)
![Screenshot 2022-02-22 at 13 59 47](https://user-images.githubusercontent.com/2019970/155149315-ebbcce55-aa78-4280-b21f-1d18e10d9c69.png)

#### Changes proposed in this Pull Request:

* adds `Margin` setting options in the `Dimensions` section of `Contact Info` block's editor
* when custom margin settings are applied by a user to the block, the block's styling reflects those changes appropriately

#### Jetpack product discussion


This PR is a part of the Expanding Jetpack Block design tooling for Full Site Editing project: pdDOJh-1P-p2 & pdDOJh-6-p2.

#### Does this pull request change what data or activity we track or use?


No, it does not.

#### Testing instructions:





1. Create a new post or a page with the `Contact info` block
2. Click ➕ icon beside the Dimensions section header in the block's editor sidebar
3. Select Margin from the dropdown menu.
4. Experiment with different values and observe whether the change applies correctly to the block's styling
5. Save the changes and review wether the margins were applied correctly in the published version of the post/page.
